osc/commandline.py

```diff
@@ -6380,6 +6380,11 @@ def parse_repoarchdescr(self, args, noinit = False, alternative_project = None,
             arg_repository, arg_arch, arg_descr = args
 
         arg_arch = arg_arch or osc.build.hostarch
+        if self.options.debug:
+            print("hostarch: ", osc.build.hostarch)
+            print("arg_arch: ", arg_arch)
+            print("arg_repository: ", arg_repository)
+            print("arg_descr: ", arg_descr)
```

Comment:
How about to print debug messages to stderr?
In my opinion: debug belongs to stderr, verbose increases regular output verbosity and belongs to stdout.

Reply:
I stole that part from here.

https://github.com/openSUSE/osc/blob/43337afdf5aa258267d407fdaf59feb3d0b15a78/osc/commandline.py#L2461-L2464

I'll make a helper to make it easier.

Reply:
Added `self_debug` helper, which prints to stderr.